### PR TITLE
Update CONTRIBUTING.md: clarify AutoRest extension deployment requirements

### PR DESCRIPTION
**Pull request by konrad-jamrozik**
As in subject.

### CODE REVIEW

**Review by weshaggard — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -110,6 +110,8 @@ If you want your changes to be deployed to [production pipeline](https://dev.azu
 - Once your PR is merged and [relevant build](https://dev.azure.com/azure-sdk/internal/_build?definitionId=1580&_a=summary) completed, approve an [npm release](https://dev.azure.com/azure-sdk/internal/_release?_a=releases&view=mine&definitionId=80) from the build.
   - Note that sometimes the npm release may report failure even when it succeeded. This is because sometimes it tries to publish package twice and succeeds only on the first try. You can verify your updated npm packages were published by reviewing your
   version is on npm. See [README `packages` section](https://github.com/Azure/azure-openapi-validator#packages). You can also look at the release build log.
+- **IMPORTANT**: changes to the AutoRest extension package require additional code updates to `openapi-alps` ADO repository, and deployeent of them. Work with this tool owner to apply these steps.
```
Comment (bdefoy) [inline CONTRIBUTING.md:113]: The azure-openapi-validator owner or the openapi-alps owner? Or are they the same
```suggestion
- **IMPORTANT**: changes to the AutoRest extension package require additional code updates to `openapi-alps` ADO repository, and deployeent of them. Work with the `openapi-alps` tool owner to apply these steps. 
```

Comment (bdefoy) [inline CONTRIBUTING.md:113]:
```suggestion
- **IMPORTANT**: changes to the AutoRest extension package require additional code updates to `openapi-alps` ADO repository, and deployment of them. Work with this tool owner to apply these steps. 
```

Comment (konrad-jamrozik) [reply CONTRIBUTING.md:113]: I actually meant this tool owner, not `openapi-alps` owner :)

Comment (konrad-jamrozik) [reply CONTRIBUTING.md:113]: https://github.com/Azure/azure-openapi-validator/pull/519

Thx for catching this :)

**Pull request merged**